includes/base/element-base.php

```diff
@@ -543,6 +543,26 @@ public function get_raw_data( $with_html_content = false ) {
 		];
 	}
 
+	public function get_data_for_save() {
```

Comment:
it reminds me the `get_settings_for_display()` thing that nobody really knows how it works...
maybe we need a better name?

Reply:
It's OK for now